datatypes/Cargo.toml

```diff
@@ -14,20 +14,20 @@ postgres = ["postgres-types", "postgres-protocol"]
 pro = ["postgres"]
 
 [dependencies]
-arrow = { version = "14.0", features = ["simd"] }
+arrow = { version = "16.0", features = ["simd"] }
 async-trait = "0.1"
 chrono = "0.4"
 float-cmp = "0.9"
 gdal = "0.12"
-geo = "0.20"
-geojson = "0.22"
+geo = "0.21"
+geojson = "0.23"
 image = "0.24"
 num = "0.4"
 num-traits = "0.2"
 ordered-float = { version= "3.0", features = ["serde"] }
 paste = "1.0"
 postgres-protocol = { version = "0.6", optional = true }
-postgres-types = { version = "0.2", features = ["derive", "with-chrono-0_4", "with-uuid-0_8"], optional = true }
+postgres-types = { version = "0.2", features = ["derive", "with-chrono-0_4", "with-uuid-0_8", "with-uuid-1"], optional = true }
```

Comment:
why do we need both?

Reply:
not yet, waiting for bb8: https://github.com/djc/bb8/issues/127

reverted it here:

[981f8ad](https://github.com/geo-engine/geoengine/pull/556/commits/981f8adf646d900b357a28eca4c49a831b459453)